src/dbus_mpris.rs

```diff
@@ -349,48 +349,21 @@ async fn create_dbus_server(
         let mv_device_name = device_name.clone();
         let sp_client = Arc::clone(&spotify_api_client);
         b.method("OpenUri", ("uri",), (), move |_, _, (uri,): (String,)| {
-            struct AnyContextId(Box<dyn PlayContextId>);
-
-            impl Id for AnyContextId {
-                fn id(&self) -> &str {
-                    self.0.id()
-                }
-
-                fn _type(&self) -> Type {
-                    self.0._type()
-                }
-
-                fn _type_static() -> Type
-                where
-                    Self: Sized,
-                {
-                    unreachable!("never called");
-                }
-
-                unsafe fn from_id_unchecked(_id: &str) -> Self
-                where
-                    Self: Sized,
-                {
-                    unreachable!("never called");
-                }
-            }
-            impl PlayContextId for AnyContextId {}
-
             enum Uri {
-                Playable(Box<dyn PlayableId>),
-                Context(AnyContextId),
+                Playable(PlayableId<'static>),
```

Comment:
Regarding these changes in `OpenUri`, there's #1145. That also simplifies this code, which can be cleaned up even more due to improvements in `rspotify`. :)

Reply:
As discussed in #1145, we'll merge that first and then I'll rebase this to remove my changes :)

Reply:
Hmm, if this is blocking on #1145, but #1145 would break builds until this is landed, we could be in an sticky situation for a period of time which is probably avoidable. 

Maybe we should have both this PR and #1145 target a non-master branch, merge both into that branch, and then squash the 2 commits and merge into master.

Reply:
Good point. Or we could just bump the MSRV in the other PR.

Reply:
> Good point. Or we could just bump the MSRV in the other PR.

Sounds like it would be the easiest. I'll do that.